src/sysmonitor/SystemMonitor.php

```diff
@@ -90,6 +94,9 @@ private function rateAndStore(SystemEvent $sysEvt)
         if ($count >= 10 && $evt instanceof RequestExceptionEvent) {
             // .. based on frequency of the same failure
             $sysEvt->severity = SystemEvent::SEVERITY_URGENT;
+        } elseif ($count >= 10 && $evt instanceof RequestStatsEvent && $evt->requestTime > 3) {
```

Comment:
I dont think we need this case either (especially not with a hardcoded 3 secs threshold).